ktlint-ruleset-standard/src/main/kotlin/com/github/shyiko/ktlint/ruleset/standard/ImportOrderingRule.kt

```diff
@@ -23,7 +24,7 @@ class ImportOrderingRule : Rule("import-ordering") {
             if (children.isNotEmpty()) {
                 val imports = children.filter { it.elementType == IMPORT_DIRECTIVE }
                 val sortedImports = imports.sortedBy { it.text }
-                if (imports != sortedImports) {
+                if (imports != sortedImports || hasTooMuchWhitespace(children)) {
                     emit(node.startOffset, "Imports must be ordered in lexicographic order", true)
```

Comment:
@gabrielittner Since you're now doing two checks here, I'd at minimum update the error message to say something about white space, or for maximum clarity, have two separate error messages. As it is, a developer would get a failure for a lexographically-ordered import list that happens to have too many white spaces, so that could be confusing.

Reply:
I changed the message to `Imports must be ordered in lexicographic order in a single group` originally, but reverted in based on Eric's comments above. I wouldn't mind adding something again.

Reply:
I don't have strong opinions about the exact language, but I do think we should be specific about what's causing a rule to fail.

If i had
```import a.b.c

import a.b.c.d
import a.b.c.d.e```

and it failed me saying that the imports had to be in lexographic order, I would feel annoyed at minimum, or at worst think the rule was broken.

Reply:
Changed the message to `Imports must be ordered in lexicographic order without any empty lines in-between`